04-overall-workflow.Rmd

```diff
@@ -142,6 +147,8 @@ git push
 
 Tags effectively label specific Git commits so you can easily find them again in future. In DevOps they show up in the Repos section under Tags. They are valuable if you've produced some outputs and wanted to label the code which produced them for future reference. If you include "/" in your tag names, they will show up in folders on DevOps.
 
+![An example of using tags within a repository.](www/images/chapter4_tags.png)
```

Comment:
To me this image looks like a product backlog image not an image of tags - let me know if you'd like to discuss the difference.